Makefile

```diff
@@ -24,18 +24,16 @@ cover:
 	go test -race -coverprofile=cover.out -coverpkg=./... ./...
 	go tool cover -html=cover.out -o cover.html
 
+# Note that installation via "go install" is not recommended
+# (https://golangci-lint.run/usage/install/#install-from-source).
+# If this causes problems, install a pre-built binary.
+#
+# When bumping the version here, then also bump the version in
+# .github/workflows/golangci-lint.yml
 $(GOLINT):
-	go install golang.org/x/lint/golint
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.51.0
 
 .PHONY: lint
 lint: $(GOLINT)
-	@rm -rf lint.log
-	@echo "Checking formatting..."
-	@gofmt -d -s $(GO_FILES) 2>&1 | tee lint.log
```

Comment:
should the gofmt / go vet / fixme checks remain?

Reply:
"govet" is enabled by default in golangci-lint. I explicitly enabled "godox" as replacement for the grep  command.

What I got wrong is "gofmt": I thought it was enabled by default., but it isn't. Will fix....

... and of course it helps to actually include the `.golangci.yml` in the commit :hot_face: